src/alpine/3.15/helix/arm32v7/Dockerfile

```diff
@@ -54,7 +54,7 @@ RUN apk update && apk add --no-cache && \
         openssl-dev \
         perl && \
     cd /tmp && \
-    git clone --depth 1 --single-branch --branch main --recursive https://github.com/dotnet/msquic && \
+    git clone --depth 1 --single-branch --branch alpine-arm32-pin-2023-08 --recursive https://github.com/dotnet/msquic && \
```

Comment:
I would probably much rather have branch that tracks MsQuic v2.2.4 (or whatever release) than some random point in time. That would also give us coverage on code used by our customers.

Reply:
We can pin to 2.2.2 (June 2023), 2.2.3+ already has the regression. Let's try